src/clients/inst_autosetup.rb

```diff
@@ -158,11 +158,9 @@ def main
       AutoinstSoftware.merge_product(AutoinstFunctions.selected_product)
 
       # configure general settings
-      AutoinstGeneral.Import(Ops.get_map(Profile.current, "general", {}))
-      Builtins.y2milestone(
-        "general: %1",
-        Ops.get_map(Profile.current, "general", {})
-      )
+      general_section = Profile.current["general"] || {}
+      AutoinstGeneral.Import(general_section)
+      log.info("general: #{general_section}")
```

Comment:
NP: What can be actually contained in the general section? Is it safe to log it? I mean, can be there any passwords or any sensitive data in general? I know, the logging was already there, so this is not a regression. I'm just asking...

Reply:
In general, it does not contain sensitive data but maybe we could log it only when debugging

https://documentation.suse.com/sles/15-SP1/html/SLES-all/configuration.html#CreateProfile-General

@imobachgs , what do you think?

Reply:
AFAIK, the general section should not contain any sensitive data (see [AutoYaST Guide](https://documentation.suse.com/sles/15-SP1/html/SLES-all/configuration.html#CreateProfile-General-example-common)).